src/components/ui/weather-card-control/weather-card-control.styled.ts

```diff
@@ -8,22 +8,29 @@ export const AddNewButton = styled.div`
   flex-direction: column;
 `;
 
-export const AddWeatherCityWrapper = styled.div``;
+export const AddWeatherCityWrapper = styled.div`
+  border-radius: 25px;
+`;
 
 export const SearchInput = styled.div`
   background: rgba(255, 255, 255, 0.26);
+  border-radius: 25px;
+  padding-left: 5px;
+
   input {
     width: 85%;
     height: 30px;
-    margin: 5px;
+    margin: 5px 3px;
     border: none;
+    border-radius: 25px;
```

Comment:
add some padding, when you type inside first symbol is cut

Reply:
ready this